pkgs/by-name/si/sile/package.nix

```diff
@@ -0,0 +1,181 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+
+  # nativeBuildInputs
+  pkg-config,
+  jq,
+  cargo,
+  rustc,
+  rustPlatform,
+
+  # buildInputs
+  lua,
+  harfbuzz,
+  icu,
+  fontconfig,
+  libiconv,
+  stylua,
+  typos,
+  darwin,
+  # FONTCONFIG_FILE
+  makeFontsConf,
+  gentium,
+
+  # passthru.tests
+  runCommand,
+  poppler_utils,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sile";
+  version = "0.15.5";
+
+  src = fetchzip {
+    url = "https://github.com/sile-typesetter/sile/releases/download/v${finalAttrs.version}/sile-${finalAttrs.version}.zip";
+    sha256 = "sha256-zP+MGCXGEg19U6tMrHIdgAAfKQT21vFtmoEROXgxUB0=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    dontConfigure = true;
+    hash = "sha256-hmgDG29C5JfQX2acMr8c3lmswa1u5XHauRWFd4QGmOo=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    jq
+    cargo
+    rustc
+    rustPlatform.cargoSetupHook
+  ];
+
+  buildInputs =
+    [
+      finalAttrs.finalPackage.passthru.luaEnv
```

Comment:
Corresponding to the `lua` argument right after the `# buildInputs` comment at the top.